gammapy/spectrum/results.py

```diff
@@ -355,161 +392,175 @@ def to_sherpa_model(self, name='default'):
 
         return model
 
-    def plot(self, ax=None, energy_unit='TeV',
-             flux_unit='cm-2 s-1 TeV-1', e_power=0, **kwargs):
-        """Plot fit function
+    def evaluate(self, x):
+        """Wrapper around astropy.modelling.models.evaluate
```

Comment:
I'd suggest this:
"""Wrapper around the ``evaluate`` method on the Astropy model classes.
"""